integration_tests/go.mod

```diff
@@ -113,5 +113,7 @@ require (
 
 replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
+	github.com/pingcap/tidb => github.com/defined2014/tidb v1.1.0-beta.0.20230712071811-87a5c15e9c7d
```

Comment:
Do you have any plans to update it?

Reply:
Yes, it think so. You could update to the master.